benchmarks/spectre_v1/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 3.5.1)
+set(CMAKE_BUILD_TYPE Release)
+
+project(safeside VERSION 0.1.0 LANGUAGES CXX)
+
+# Use C++11 without extensions
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_EXTENSIONS off)
+
+add_executable(unmitigated microbenchmark.cc)
+target_compile_options(unmitigated PUBLIC -O0)
```

Comment:
We can probably leave out optimization flags; they're provided at toplevel based on the kind of build. The important thing here is that the *same* flags are used for the unmitigated and mitigated versions, which should be true as long as we don't override them.

Reply:
Ah, nevermind, I think I understand -- you're using `-O0` to prevent the compiler from optimizing away the entire benchmark, since the array manipulation doesn't actually have visible effects.

That's okay for now, but please leave a comment explaining what we're up to.

Later, we can replace this with tricks like [forcing the compiler to compute values eagerly](https://github.com/travisdowns/zero-fill-bench/blob/c30d8fc88d3f560ba165ef756167a2a703678134/opt-control.hpp#L11-L36).

Reply:
I added the comment and FIXME. Would it be better to add a github issue for this? What's the convention?

Reply:
FIXME seems fine.